exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -0,0 +1,275 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public final class RobotTest {
+
+    @Test
+    public void testRobotIsCreatedWithCorrectInitialPosition() {
+        final GridPosition initialGridPosition = new GridPosition(0, 0);
+        final Robot robot = new Robot(initialGridPosition, Orientation.NORTH);
+
+        final GridPosition actualGridPosition = robot.getGridPosition();
+
+        assertTrue(actualGridPosition.equals(initialGridPosition));
```

Comment:
What are your thoughts on using `assertEquals()`?  This increases the fidelity of signal from the test system: the programmer can know what's wrong just looking at the failure message.